### DISCUSSION TITLE
Install on Windows Server

### DISCUSSION THREAD

**Discussion by PIMEMenorca** [globaleaks/globaleaks-whistleblowing-software#3868] [Q&A] [answered]
Hello,

I would like to install GlobaLeaks but my server is running Windows Server OS. Is there any option to install it without using virtual machines. I tried using Windows Subsystem Linux but I get: Ouch! The installation is complete but GlobaLeaks failed to start.

Thanks in advance!

**Comment by evilaliv3** [comment, marked as answer]
Hello, this is actually not possible at the moment unfortunately. We always have considered such an option but the community considered to keep focused on a single platform.
